lighthouse-core/gather/gatherers/dobetterweb/appcache.js

```diff
@@ -10,8 +10,11 @@ const FRGatherer = require('../../../fraggle-rock/gather/base-gatherer.js');
 /* global document */
 
 class AppCacheManifest extends FRGatherer {
+  static GathererSymbol = Symbol('AppCacheManifest');
```

Comment:
define the symbol both on the class and on the instance's meta for use by the dependent gatherers and config respectively

Reply:
i don't care a lot, but i'd expose this as `symbol` rather than `GathererSymbol`. consistent and they both point to the same thing anyway.